lib/mayaUsd/render/vp2RenderDelegate/draw_item.h

```diff
@@ -89,21 +91,44 @@ class HdVP2DrawItem final : public HdDrawItem
 
     ~HdVP2DrawItem();
 
+    void AddRenderItem(MHWRender::MRenderItem* item, const HdGeomSubset* geomSubset = nullptr);
+
+    using RenderItemDataVector = std::vector<RenderItemData>;
+    RenderItemDataVector& GetRenderItems() { return _renderItems; }
+
     /*! \brief  Get access to render item data.
      */
-    RenderItemData& GetRenderItemData() { return _renderItemData; }
+    RenderItemData& GetRenderItemData()
```

Comment:
I left all of these convenience methods around so that I wouldn't have to make as many changes to basisCurves.h